src/Solver/time_stepping/TimeCluster.h

```diff
@@ -895,7 +898,8 @@ class seissol::time_stepping::TimeCluster : public seissol::time_stepping::Abstr
       dr::output::OutputManager* i_faultOutputManager,
       seissol::SeisSol& seissolInstance,
       LoopStatistics* i_loopStatistics,
-      ActorStateStatistics* actorStateStatistics);
+      ActorStateStatistics* actorStateStatistics,
+      seissol::initializer::parameters::DamagedElasticParameters* damageparameters);
```

Comment:
No need to pass `damageparameters`, can be accessed through `seissolInstance.getSeisSolParameters()...`